src/app/service/categories/ServiceCategories.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import * as actions from '../../../actions';
+import Header from '../../../components/header';
+import Accordion from '../../../components/accordion';
+import Selector from '../../../components/selector';
+import Button from '../../../components/button';
+
+import NavBar from '../../NavBar';
+
+const LoadingView = () => (
+  <div className="d-flex flex-column">
+    <NavBar title="Services info" />
+    <p>
+      <i className="fa fa-spinner fa-spin" aria-hidden="true" /> Loading location data ...{' '}
+    </p>
+  </div>
+);
+
+class ServiceCategories extends Component {
+  state = { active: 0, selected: {} };
+
+  componentWillMount() {
+    if (!this.props.taxonomy) {
+      this.props.getTaxonomy();
+    }
+  }
+
+  onToggleOpen = value =>
+    this.setState(({ active }) => ({ active: active !== value ? value : -1 }));
+
+  onSelect = (category) => {
+    const { selected } = this.state;
+    const selection = selected[category.id];
+    this.setState({ selected: { ...selected, [category.id]: !selection } });
+  };
+
+  onSubmit = () => {
+    const { selected } = this.state;
+    const { locationId } = this.props.match.params;
+    const locationServices = this.props.taxonomy.filter(service => selected[service.id]);
+    this.props.selectCategories(locationId, locationServices);
+    this.props.history.push(`/location/${locationId}/services/recap`);
+  };
+
+  render() {
+    const { active, selected } = this.state;
+    const { taxonomy } = this.props;
+
+    if (!taxonomy) {
+      return <LoadingView />;
+    }
+
+    return (
+      <div className="text-left">
+        <NavBar title="Services info" />
+        <div className="mb-5">
+          <div className="py-5 px-3 container">
+            <Header>What programs and services are available at this location?</Header>
+          </div>
+          <Accordion>
+            <Accordion.Item
+              active={active === 0}
+              onClick={() => this.onToggleOpen(0)}
+              title="All Services"
+              icon="home"
+            />
+            <Accordion.Content active={active === 0}>
```

Comment:
I'm not entirely sure I follow what's happening here.
IIUC, there's only 1 accordion, with a selector containing an option for each taxonomy item at the first level? Isn't it supposed to be 1 accordion for each taxonomy item at the first level, with a selector containing options for each of its child taxonomies? Currently, I don't see where `category.children` are used.

Also, I'd expect that rather than "Add another shelter service", we'd have something like `Add another ${category.name} service`, as I don't recall anything special about shelter specifically in the designs.

Reply:
Btw, if you're testing this with a local server (and not the dev RDS), it might help to run `DELETE FROM taxonomies;` and `DELETE FROM services;`, and then `sequelize db:seed:all` in streetlives-api (assuming you have the sequelize CLI installed).
That would seed the Streetlives taxonomy in the DB, instead of the Connections NYPL one.

Reply:
Ah understood. I wasn't seeing this updated data so I didn't know how best to render it. I followed your steps and updated this PR to dynamically render this data.